### PR TITLE
Game history bubble

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

**Comment by dionyziz** [comment]
Pair programming @VitSalis @dionyziz. @pkakelas please review.

File: views/footer.php [hunk 1]
```diff
@@ -9,5 +9,6 @@
                 </li>
             </ul>    
         </footer>
+        <script type="text/javascript" src="static/script/game/view.js"></script>
```
Comment (pkakelas) [inline views/footer.php]: I suggest you to move this view to header.php where all other scripts/styles are placed so we can have them in one file. You can use $( document ).ready(); and move your code inside ready(), in order to ensure that DOM will be loaded before the script gets executed.

File: static/style/game/view.css [hunk 2]
```diff
@@ -142,3 +146,61 @@ aside ol li del {
...
+    color: rgba(0, 0, 0, 0.7);
+    position: absolute;
+    bottom: -15px;
+    right: 13px;
+}
+.infobubble h3 {
+    color: white;
+    margin: 0;
+    padding: 0 0 15px 0;
+    font-size: 100%;
```
Comment (pkakelas) [inline static/style/game/view.css:170]: Why is that necessary?

File: static/style/game/view.css [hunk 3]
```diff
@@ -142,3 +146,61 @@ aside ol li del {
 aside ol li.you {
     background-color: #ffffca;
 }
+.infobubble {
+    z-index: 1001;
```
Comment (pkakelas) [inline static/style/game/view.css:150]: Why 1001? Do we have an element with a lower stack order?

File: static/style/game/view.css (hunk 2 above)
Comment (dionyziz) [inline static/style/game/view.css:170]: screen shot 2014-04-02 at 4 48 21 pm
Default h3 font size is larger than 100%

File: static/style/game/view.css (hunk 3 above)
Comment (dionyziz) [inline static/style/game/view.css:150]: Yes .navbar has z-index of 1000

**Comment by pkakelas** [comment]
Pls take screenshot for the description of the pull request

**Comment by dionyziz** [comment]
screen shot 2014-04-02 at 8 50 14 pm

**Comment by dionyziz** [comment]
And video for the animation:

https://www.youtube.com/watch?v=5WlDNwxlnH0

**Comment by vitsalis** [comment]
Why is this not merged?

**Comment by dionyziz** [comment]
Waiting for @pkakelas to merge

**Comment by vitsalis** [comment]
Actually, don't merge. I haven't fixed the bug with the bubble appearing under the creature yet.

**Comment by pkakelas** [comment]
LGTM. Vit when you are ready, I'll merge you
